Config.cmake.in

```diff
@@ -1,5 +1,11 @@
 @PACKAGE_INIT@
 
+# The exported target depends on targets from these dependencies,
+# so we need to find them first.
+include(CMakeFindDependencyMacro)
+find_dependency(ZLIB)
+find_dependency(JPEG)
```

Comment:
Question: how does this deal with zlib and jpeg being dynamically linked?
In that case the application that is consuming IrrlichtMt does not have to worry about them at all.

(The public headers use nothing from libjpeg, zlib or libpng. I think you know that already.)

Reply:
find_dependency() has nothing to do with the  application consuming IrrlichtMt. IrrlichtMtTargets can link dependencies as targets, and find_dependency gets the target so it exists when IrrlichtMtTargets is included. But actually these are probably completely unnecessary for JPEG because it should be private. ZLib should be public(?) and if I link it as a target this will probably be needed. How CMake handles transitive dependencies that are dynamically linked, I have no idea.